build.gradle

```diff
@@ -2,12 +2,12 @@ apply plugin: 'com.github.ben-manes.versions'
 
 buildscript {
   ext.versions = [
-    kotlin: '1.3.50',
+    kotlin: '1.3.60',
```

Comment:
required to avoid weird kapt build issues

```
FAILURE: Build failed with an exception.
* What went wrong:
Execution failed for task ':paparazzi:kaptKotlin'.
> Could not resolve all files for configuration ':paparazzi:_classStructurekaptKotlin'.
   > Could not resolve com.squareup.okio:okio:2.4.2.
     Required by:
         project :paparazzi
      > Cannot choose between the following variants of com.squareup.okio:okio:2.4.2:
          - jvm-api
          - jvm-runtime
          - metadata-api
        All of them match the consumer attributes:
          - Variant 'jvm-api' capability com.squareup.okio:okio:2.4.2:
...
```